cocktails.web/src/organisims/AccountNavigationBar/AccountNavigationBar.tsx

```
@@ -12,66 +12,70 @@ import ArticleIcon from '@mui/icons-material/Article';
-                        <b>Community Starndards and Policies</b>
-                    </Typography>
-                    <Typography gutterBottom sx={{ color: 'text.secondary', fontSize: 16 }}>
-                        Read the policies and standards in place while navigating and interacting on Cezzis.com.
-                    </Typography>
-                    <ProfileSettingItemLink icon={<ArticleIcon />} text='Terms of Service' testId='p-settings-terms' navigatePath='/account/policies/terms-of-service' />
-                    <ProfileSettingItemLink icon={<LockPersonIcon />} text='Privacy Policy' testId='p-settings-privacy' navigatePath='/account/policies/privacy-policy' />
-                    <ProfileSettingItemLink icon={<CookieIcon />} text='Cookie Preferences' testId='p-settings-coockies' navigatePath='#' onClick={(e) => showCookieBot(e)} />
-                </CardContent>
-            </Card>
+const AccountNavigationBar = ({ testId, fullScreen = false }: AccountNavigationBarProps) => {
+    const { } = useAuth0();
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)